### ISSUE TITLE
Issue with "ARIA: none role": remove all children from the accessibility tree? or remove semantics of one element?

### ISSUE THREAD

**Issue by 1j01** [mdn/content#11754]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/none_role

#### What information was incorrect, unhelpful, or incomplete?

It says the `none`/`presentation` roles remove the element and its descendants from the accessibility tree. This implies it hides all children from screen readers.

#### What did you expect to see?

It should be clear and consistent with [Using the presentation role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role), which describes it as removing semantics of the element and in some circumstances certain *related* children, like table rows for a table. (Is this just for children with roles that are valid only with certain parent element roles?)

In particular, it should be clear [how it's different from `aria-hidden`](https://www.scottohara.me/blog/2018/05/05/hidden-vs-none.html)

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/none_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/none_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/none_role/index.md
* Last commit: https://github.com/mdn/content/commit/079c1c447e3e14f0d395b2963e0851dc93c80e37
* Document last modified: 2021-11-08T01:10:30.000Z

</details>
